### PR TITLE
docs: update sidebar style of website 

### PR DESCRIPTION
**Pull request by li-jia-nan**
[[中文版模板 / Chinese template](https://github.com/ant-design/ant-design/blob/master/.github/PULL_REQUEST_TEMPLATE/pr_cn.md?plain=1)]

### 🤔 This is a ...

- [ ] New feature
- [ ] Bug fix
- [x] Site / documentation update
- [ ] Demo update
- [ ] Component style update
- [ ] TypeScript definition update
- [ ] Bundle size optimization
- [ ] Performance optimization
- [ ] Enhancement feature
- [ ] Internationalization
- [ ] Refactoring
- [ ] Code style optimization
- [ ] Test Case
- [ ] Branch merge
- [ ] Workflow
- [ ] Other (about what?)

### 📝 Changelog

| Language   | Changelog |
| ---------- | --------- |
| 🇺🇸 English | docs: update GetRef/GetProp/GetProps docs |
| 🇨🇳 Chinese | docs: update GetRef/GetProp/GetProps docs |

### ☑️ Self-Check before Merge

⚠️ Please check all items below before requesting a reviewing. ⚠️

- [x] Doc is updated/provided or not needed
- [x] Demo is updated/provided or not needed
- [x] TypeScript definition is updated/provided or not needed
- [x] Changelog is provided or not needed

### CODE REVIEW

**Review by zombieJ — DISMISSED** [review]

**Review by kiner-tang — DISMISSED** [review]

File: components/_util/index.en-US.md [hunk 1]
```diff
@@ -12,12 +12,15 @@ group:
   order: 99
 ---
 
+Available since `5.13.0`.
```
Comment (afc163) [inline components/_util/index.en-US.md:15]: 1. 可以把这个 New 改成版本号。
图片
2. 可以这个地方增加版本号。
图片
3. 配置在上面的 meta 信息里。

Comment (li-jia-nan) [reply components/_util/index.en-US.md:15]: 可以，我来做一下

File: components/_util/index.en-US.md [hunk 2]
```diff
@@ -5,19 +5,23 @@ description: Utilities are used to assist development and provide some common ut
...
 coverDark: https://mdn.alipayobjects.com/huamei_7uahnr/afts/img/A*rRDlT7ST8DUAAAAAAAAAAAAADrJ8AQ/original
+tag: 5.13.0
 demo:
   cols: 2
 group:
   title: Other
   order: 99
 ---
 
+Available since `5.13.0`.
```
Comment (afc163) [inline components/_util/index.en-US.md:16]: 类似这样的语句都可以去掉了。

Comment (li-jia-nan) [reply components/_util/index.en-US.md:16]: 我觉得留着比较好，有些用户眼神不好使😕

**Review by crazyair — APPROVED** [review]

File: components/watermark/index.zh-CN.md [hunk 3]
```diff
@@ -8,10 +8,10 @@ cover: https://mdn.alipayobjects.com/huamei_7uahnr/afts/img/A*wr1ISY50SyYAAAAAAA
 coverDark: https://mdn.alipayobjects.com/huamei_7uahnr/afts/img/A*duAQQbjHlHQAAAAAAAAAAAAADrJ8AQ/original
 demo:
   cols: 1
-tag: New
+tag: 5.1.0
 ---
 
-自 `5.0.0` 版本开始提供该组件。
+自 `5.1.0` 版本开始提供该组件。
```
Comment (afc163) [inline components/watermark/index.zh-CN.md:14]: 这个文字不需要了，在上面的 meta 里直接展示

```
版本：自 `x.y.z` 后支持
```


```
Version: supported in `x.y.z`
```

Comment (afc163) [reply components/watermark/index.zh-CN.md:14]: 图片

Comment (li-jia-nan) [reply components/watermark/index.zh-CN.md:14]: 我再开个 pr

Comment (li-jia-nan) [reply components/watermark/index.zh-CN.md:14]: - #48226

**Pull request merged**